samples/hosting/docker/Core_7/Receiver/Program.cs

```diff
@@ -19,6 +20,12 @@ static IHostBuilder CreateHostBuilder(string[] args)
         {
             return Host.CreateDefaultBuilder(args)
                 .UseConsoleLifetime()
+                .ConfigureLogging(logging =>
+                {
+                    logging.AddConsole();
+                    logging.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Information);
```

Comment:
I guess it doens't matter much but I would just add a using above and then use `LogLevel.Information`

Reply:
It was clashing the the nsb logging namespace but I fixed it by removing the critical error action